internal/migrations/v2.5.0.go

```diff
@@ -11,7 +11,7 @@ func V2_5_0(db *sqlx.DB, fs stuffbin.FileSystem, ko *koanf.Koanf) error {
 	// Insert new preference settings.
 	if _, err := db.Exec(`
 		INSERT INTO settings (key, value) VALUES
- 			('app.enable_rss_content', 'true')
+ 			('app.enable_public_archive_rss_content', 'false')
```

Comment:
You've changed this to `false`, but in `schema.sql` we have:
```
('app.enable_public_archive_rss_content', 'true')
```

Maybe it's intentional to give different defaults when installing fresh vs upgrading?

Reply:
Ah yes, forgot to document this. The existing installations that use the archive feature would expect the content not to be published (as there's no support now). We don't want to break this expectation+behaviour by publishing full content after an upgrade. This should be an explicit choice for existing installation.

Reply:
@knadh agreed, that’s reasonable decision.